piq/gmsd.py

```diff
@@ -179,9 +181,14 @@ def multi_scale_gmsd(prediction: torch.Tensor, target: torch.Tensor, data_range:
     Returns:
         Value of MS-GMSD. 0 <= GMSD loss <= 1.
     """
-    _validate_input(input_tensors=(prediction, target), allow_5d=False, scale_weights=scale_weights)
+    _validate_input(
+        input_tensors=(prediction, target), allow_5d=False, scale_weights=scale_weights, data_range=data_range)
     prediction, target = _adjust_dimensions(input_tensors=(prediction, target))
-
+    
+    # Rescale to [0, 255] range
```

Comment:
Same as above. It is not guaranteed that the result will be in the [0, 255] range.